src/chat/chat.js

```diff
@@ -1,115 +1,26 @@
 import Config from "../config/config.js"
+import Message, { NotifMessage, EventMessage } from "../config/message.js";
```

Comment:
```suggestion
import { NotifMessage, EventMessage } from "../config/message.js";
```
Don't import things you aren't using.